models/articles.models.js

```diff
@@ -25,3 +25,17 @@ exports.fetchAllArticles = () => {
       return results.rows;
     });
 };
+
+exports.insertNewComment = (article_id, body, username) => {
+  if (!username || !body || !article_id) {
```

Comment:
You may not need to handle this error manually, are these required fields on the comments table? If so and they are not provided then psql will throw and error for you.